src/main/java/com/xatkit/core/recognition/processor/PerspectiveapiInterface.java

```diff
@@ -0,0 +1,281 @@
+     * @param clientToken      the client token
+     * @param sessionId        the session id
+     * @param attributesPrefix the attributes prefix for the keys of the map they will be stored in
+     */
+    PerspectiveapiInterface(@NonNull String apiKey, String language, Boolean doNotStore, String clientToken,
+                            String sessionId, String attributesPrefix) {
+
+        this.apiKey = apiKey;
+        this.language = language != null ? language : "en";
+        this.doNotStore = doNotStore != null ? doNotStore : false;
+        this.clientToken = clientToken;
+        this.sessionId = sessionId;
+        this.attributesPrefix = attributesPrefix;
+        languageAttributes = new HashMap<>();
+        languageAttributes.put("en", new AttributeType[] {
+                AttributeType.TOXICITY,
+                AttributeType.SEVERE_TOXICITY,
+                AttributeType.TOXICITY_FAST,
+                AttributeType.IDENTITY_ATTACK,
+                AttributeType.INSULT,
+                AttributeType.PROFANITY,
+                AttributeType.THREAT,
+                AttributeType.SEXUALLY_EXPLICIT,
+                AttributeType.FLIRTATION,
+                AttributeType.ATTACK_ON_AUTHOR,
+                AttributeType.ATTACK_ON_COMMENTER,
+                AttributeType.INCOHERENT,
+                AttributeType.INFLAMMATORY,
+                AttributeType.LIKELY_TO_REJECT,
+                AttributeType.OBSCENE,
+                AttributeType.SPAM,
+                AttributeType.UNSUBSTANTIAL
+        });
+        languageAttributes.put("es", new AttributeType[] {
+                AttributeType.TOXICITY,
+                AttributeType.SEVERE_TOXICITY,
+                AttributeType.IDENTITY_ATTACK_EXPERIMENTAL,
+                AttributeType.INSULT_EXPERIMENTAL,
+                AttributeType.PROFANITY_EXPERIMENTAL,
+                AttributeType.THREAT_EXPERIMENTAL
+        });
+        this.setRequestedAttributes();
+    }
+
+    /**
+     * Sets the attributes available for the chosen language
+     */
+    private AttributeType[] setRequestedAttributes() {
+        if (language.equals("es")) {
+            requestedAttributes = languageAttributes.get("es");
+        }
+        else {
+            requestedAttributes = languageAttributes.get("en");
+        }
+        return requestedAttributes;
+    }
+
+    /**
+     * Sets the comment that will be analyzed
+     */
+    private String setCommentText(String commentText) {
+        this.commentText = commentText;
+        return commentText;
+    }
+
+    /**
+     * Analyzes the comment and returns the punctuation for each attribute
+     *
+     * @param comment the comment
+     * @return the hash map with attributes and their respective scores
+     */
+    public HashMap<String, Double> analyzeRequest(String comment) {
+        JSONObject response = new JSONObject();
+        HashMap<String, Double> scores = new HashMap<>();
```

Comment:
For inline attributes use this pattern:
```java
Map<String, Double> scores = new HashMap<>();
```
Similar to the comment above: the rest of your code does not need to know what kind of map you are using. This will make things easier if you want to change the map implementation.

> 📚 The rest of the code still needs to know that you are using a `Map`. You cannot use a more generic type like `Collection` because it does not provide the `put/get` method.